integrations/tensorflow/test/python/iree_tfl_tests/test_util.py

```diff
@@ -63,14 +63,14 @@ def setUp(self):
       return
     self.workdir = _setup_artifacts_dir("download")
     print(f"TMPDIR = {self.workdir}")
-    self.tflite_file = '/'.join([self.workdir, 'model.tflite'])
-    self.tflite_ir = '/'.join([self.workdir, 'tflite.mlir'])
-    self.iree_ir = '/'.join([self.workdir, 'tosa.mlir'])
+    self.tflite_file = '/'.join([self.workdir, 'model.bc'])
```

Comment:
is the canonical mlir extension bc? (I hope not, as that's llvm's extension!)

Reply:
Fixed in followup